doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
-            auto counter = reinterpret_cast<const CounterTopic*>(sample.get());
-            /* process the received data using counter */
-        })
-        .if_empty([] { /* no data received but also no error */ })
-        .or_else([](iox::popo::ChunkReceiveResult) { /* handle the error */ });
-}
-```
+Once it has offered its topic, it is able to publish (send) data of the specific type. Note that the default is to
+have multiple publishers for the same topic (n:m communication). A compile-time option to restrict iceoryx to
+1:n communication is available. Should 1:n communication be used RouDi checks for multiple publishers on the same 
+topics and raises an error if there is more than one publisher for a topic.
 
-Note that since the received sample received is untyped (``iox::popo::Sample<const void>``), we cannot use
-``operator->`` to access the members of the underlying type but have to cast it to the correct type ``CounterTopic``
-manually. We know this type since it is uniquely identified by the topic we subscribed to (in our case ``CounterTopic``). 
-A ``reinterpret_cast`` is used to interpret the data as a ``CounterTopic``. Note that a ``static_cast`` would also work here,
-but a ``reinterpret_cast`` is used to emphasize the data-agonstic nature of the data transmission itself. 
+## Subscriber
 
-As in the untyped case we also could use a loop to get all samples as long as they are available.
+Symmetrically a subscriber also corresponds to a topic and thus needs a service description to be constructed. As for
+publishers we distinguish between typed and untyped subscribers.
 
-A non-functional approach is also possible but more verbose.
+Once a subscriber is subscribed to some topic, it is able to receive data of the type tied to this topic. In the
+untyped case this is raw memory and the user must take care that it is interpreted in a way that is compatible to the
+data that was actually sent.
 
-```cpp
-while (keepRunning)
-    {
-        // wait for new data (either sleep and wake up periodically or by notification from the waitset)
-
-        auto result = subscriber->take();
-
-        if(!result.has_error()) 
-        {
-            auto& maybeSample = result.value();
-            if (maybeSample.has_value())
-            {
-                auto& sample = maybeSample.value();
-                void* chunk = sample.get();
-
-                //interpret and process the data
-                const CounterTopic* counter = reinterpret_cast<const CounterTopic*>(chunk);
-            }
-            else
-            {
-                // we received no data
-            }
-        } else {
-            iox::popo::ChunkReceiveResult& error = result.get_error();
-            //handle the error
-        }
-    }
-```
+When multiple publishers have offered the same topic the subscriber will receive the data of all of them (but in
+indeterminate order between different publishers).
 
-### Shutdown
+## Waitset
```

Comment:
The overview should provide an overview. The WaitSet and Listener are constructs that should be contained in that overview otherwise it is not an overview.

What kind of ASCII art would you like to see here? A class diagram? I am a little bit puzzled.

Reply:
What I have in mind is something like the [GIF](https://user-images.githubusercontent.com/8661268/74612998-b962bc80-510a-11ea-97f0-62f41c5d287b.gif) but only static.

Reply:
@mossmaurice As mentioned before, any graphic will be part of a follow-up PR since it might not be finished until tomorrow. 
Regarding the Waitset & Listener I agree with @elfenpiff

Reply:
If you are ok with it, I'd not rush here and take some more time to clarify these questions. Quality over quantity IMHO. Besides that, the final documentation is not needed for tomorrow's v1.0-rc1

Reply:
@mossmaurice I added your suggestions to the issue.